### PR TITLE
Inline Clock Svgs

### PR DESCRIPTION
**Pull request by NataliaLKB**
Reusing one clock image, instead of having three. 

Also, deleted `common/app/views/fragments/dateline.scala.html` as I doesn't seem to be used anywhere.

Here is an example of this icon used:

image

### CODE REVIEW

File: static/src/stylesheets/base/_inline-svgs.scss [hunk 1]
```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon.light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```
Comment (johnduffell) [inline static/src/stylesheets/base/_inline-svgs.scss]: is this . separator correct for BEM syntax?

**Comment by johnduffell** [comment]
screenshots can sometimes help for this kind of thing just to illustrate what the icons did/will look like, but I think I know which ones you're changing here
apart from the BEM query :+1:

File: static/src/stylesheets/base/_inline-svgs.scss (hunk 1 above)
Comment (NataliaLKB) [inline static/src/stylesheets/base/_inline-svgs.scss]: @johnduffell ah! Would `.inline-icon.inline-icon--light-grey` be better?

**Comment by NataliaLKB** [comment]
@johnduffell Thanks for taking a look! 

I do hope we don't see any differences with the icons on the site. They should remain the same as now, simply inline svgs instead of css data images. 

These changes are being done to reduce the size of the css loading onto the site.

File: static/src/stylesheets/base/_inline-svgs.scss (hunk 1 above)
Comment (davidrapson) [inline static/src/stylesheets/base/_inline-svgs.scss]: `.inline-icon--light-grey` would be a typical BEM selector, you wouldn't normally chain the selector under BEM conventions.

**Comment by paperboyo** [comment]
@NataliaLKB Great work! Most of them should already be optimised to the bone, but if you notice any that aren't (e.g. have more than 1 decimal place) - let me know. I'll butcher them.

_85337456_chris_the_sheep

File: static/src/stylesheets/base/_inline-svgs.scss (hunk 1 above)
Comment (NataliaLKB) [inline static/src/stylesheets/base/_inline-svgs.scss]: @davidrapson Thanks that is helpful! I will make that change.

Comment (davidrapson) [inline static/src/stylesheets/base/_inline-svgs.scss]: @NataliaLKB If you've not seen it already http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ is good, as the naming conventions there are the closest to the ones used on theguardian.com

Comment (NataliaLKB) [inline static/src/stylesheets/base/_inline-svgs.scss]: @davidrapson great! I will take a look.

**Comment by NataliaLKB** [comment]
Haha thanks @paperboyo

**Comment by NataliaLKB** [comment]
Issues fixed, and tests passing. Merging!

File: static/src/stylesheets/base/_inline-svgs.scss [hunk 2]
```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon--light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```
Comment (johnduffell) [inline static/src/stylesheets/base/_inline-svgs.scss]: sorry I missed that whole PR going out, I think the same applies to the .small as the .light-grey about BEM syntax

Comment (NataliaLKB) [inline static/src/stylesheets/base/_inline-svgs.scss]: Yes that makes sense. This is a different style guide to what I am used to so thanks so much for checking. I will submit a new PR with this change.

**Pull request merged**